packages/suite/src/components/suite/FormattedCryptoAmount/index.tsx

```diff
@@ -52,23 +57,23 @@ export const FormattedCryptoAmount = ({
 
     const areSatsSupported = !!networkFeatures?.includes('amount-unit');
 
-    let formattedValue = value;
+    let formattedValue = Number(value);
```

Comment:
the name is confusing, it is not formatted value, right? it's more like parsed value

Reply:
Named it that way because it can also be converted to satoshis later. So it's not just parsed :( What name would you prefer?

Reply:
There is conversion to Number even if isSatoshis, so maybe `valueNumber`?